src/main/java/io/vertx/core/json/jackson/DatabindCodec.java

```diff
@@ -27,29 +27,28 @@
 import java.io.InputStream;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
 
 /**
  * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
  */
 public class DatabindCodec extends JacksonCodec {
 
   private static final ObjectMapper mapper = new ObjectMapper();
-  private static final ObjectMapper prettyMapper = new ObjectMapper();
+  private static final AtomicReference<ObjectMapper> prettyMapper = new AtomicReference<>();
```

Comment:
I do not understand why we need atomic here ? what is the purpose of this ?

Reply:
what is the issue with not lazy creating the pretty mapper ?

Reply:
yes I do prefer, I don't think we need to save anything here I prefer to use atomic to be used for what they are designed for, if we really need some kind of lazy loading we would use the class lazy loading trick instead

Reply:
Done